### DISCUSSION TITLE
Batch Effect Correction in ATAC `data` slot

### DISCUSSION THREAD

**Discussion by Chhiring-Lama** [stuart-lab/signac#971] [Q&A] [answered]
Hi, 

I am running peak-gene linkage in my integrated seurat object (which contain scRNA-seq and scATAC-seq data). It was integrated using SCT assay, in addition to using Harmony to batch correct the ATAC data. However, since I am using the `data` slot from the gene expression and chromatin assay which remain unchanged, I am afraid my analysis will still suffer from batch effect. How can I account for batch effect while running `LinkPeaks()`?

**Comment by timoast** [comment, marked as answer]
`LinkPeaks()` currently does not take into account batch differences. We could potentially use a GLM with batch as a latent variable to account for batch differences, but implementing this is not a high priority right now. If there are substantial batch differences between datasets I'd suggest running `LinkPeaks()` separately in the different batches and then combining the results.
